src/lib.rs

```diff
@@ -57,10 +57,7 @@ pub use bytemuck::Pod;
 pub use bytemuck::Zeroable;
 
 pub use crate::internal::convert::*;
-pub use crate::internal::ops::*;
 pub use crate::internal::pixel::*;
-pub use crate::internal::rgb::*;
```

Comment:
Isn't this a breaking change?

Reply:
Not really, `rgb` module doesn't contain any exportable items, only `impl`s and non-export macros.